src/View/mainpage/IsodoseTab.py

```diff
@@ -33,21 +35,22 @@ def init_layout(self):
 
     def init_color_isod(self):
         """
-        Create a dictionary containing the colors for each isodose.
+        Create a list containing the colors for each isodose.
 
-        :return: Dictionary where the key is the percentage of isodose and the value a QColor object.
+        :return: List of QColor objects that correspond to each isodose as defined outside the class..
```

Comment:
This is still a Dictionary

Reply:
Whoops, I changed the type to a list while testing, and then changed it back to a dictionary but forgot to update the documentation. Will fix.